linux_os/guide/system/software/integrity/software-integrity/aide/aide_periodic_cron_checking/rule.yml

```diff
@@ -7,9 +7,17 @@ title: 'Configure Periodic Execution of AIDE'
 description: |-
     At a minimum, AIDE should be configured to run a weekly scan.
     To implement a daily execution of AIDE at 4:05am using cron, add the following line to <tt>/etc/crontab</tt>:
-    <pre>05 4 * * * root {{{ aide_path }}} --check</pre>
+    {{%- if 'ubuntu' not in product %}}
+    <pre>05 4 * * * root {{{ aide_bin_path }}} --check</pre>
+    {{%- else %}}
+    <pre>05 4 * * * root /usr/bin/aide.wrapper --config /etc/aide/aide.conf --check</pre>
+    {{%- endif %}}
     To implement a weekly execution of AIDE at 4:05am using cron, add the following line to <tt>/etc/crontab</tt>:
-    <pre>05 4 * * 0 root {{{ aide_path }}} --check</pre>
+    {{%- if 'ubuntu' not in product %}}
+    <pre>05 4 * * 0 root {{{ aide_bin_path }}} --check</pre>
+    {{%- else %}}
+    <pre>05 4 * * 0 root /usr/bin/aide.wrapper --config /etc/aide/aide.conf --check</pre>
```

Comment:
Why the variables aide_conf_path and aide_bin_path aren't used here?

Reply:
aide_conf_path can probably be used... but the aide_bin_path doesn't contain the correct value `aide.wrapper` because other rules already use this variable in a different context... there might be the need to instantiate multiple variables. Briefly checking these aide binaries/wrappers it looks like they differ a lot from distro to distro.